bundles/oms-retry-capture-process/src/FondOfKudu/Zed/OmsRetryCaptureProcess/Communication/Plugin/Condition/RetryCaptureTimeoutConditionPlugin.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace FondOfKudu\Zed\OmsRetryCaptureProcess\Communication\Plugin\Condition;
+
+use DateTime;
+use Orm\Zed\Sales\Persistence\SpySalesOrderItem;
+use Propel\Runtime\Exception\PropelException;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\Oms\Dependency\Plugin\Condition\ConditionInterface;
+
+/**
+ * @method \FondOfKudu\Zed\OmsRetryCaptureProcess\OmsRetryCaptureProcessConfig getConfig()
+ */
+class RetryCaptureTimeoutConditionPlugin extends AbstractPlugin implements ConditionInterface
+{
+    /**
+     * @param \Orm\Zed\Sales\Persistence\SpySalesOrderItem $orderItem
+     *
+     * @return bool
+     */
+    public function check(SpySalesOrderItem $orderItem): bool
+    {
+        try {
+            $createdAt = $orderItem->getOrder()->getCreatedAt();
+            $interval = $createdAt->diff(new DateTime());
+        } catch (PropelException $e) {
+            return false;
+        }
+
+        return $interval->h <= $this->getConfig()->getHoursAfterCaptureFinalFailed();
```

Comment:
Should this return true if interval is bigger than the configured time? Right now it send's true right

Reply:
Do I have a thinking error here? The condition should return 'true' as long as the order is not older than the configured time (default 12 hours).

Reply:
But if it's a timeout shouldn't it be true when the order is older than 12 hours? 
Like: 
"Check if timeout is true"

That's why I suggested to change the name to something like "ShouldBreak..." or something similar so that's easier to understand.